lib/lit/i18n_backend.rb

```diff
@@ -129,5 +131,12 @@ def valid_locale?(locale)
       locales = ::Rails.configuration.i18n.available_locales
       !locales || locales.map(&:to_s).include?(locale.to_s)
     end
+
+    def is_ignored_key(key_without_locale)
+      Lit.ignored_keys.each do |k|
```

Comment:
You could use `any?`

    Lit.ignored_keys.any?{ |k| key_without_locale.start_with?(k) }